charmcraft/commands/init.py

```diff
@@ -90,8 +89,12 @@ def run(self, args):
 
         if args.author is None:
             try:
+                # Not all platforms support pwd, so we do a late import here and handle
+                # the potential import error.
+                import pwd
+
                 gecos = pwd.getpwuid(os.getuid()).pw_gecos.split(",", 1)[0]
-            except KeyError:
+            except (KeyError, ImportError):
```

Comment:
I don't like doing the import here in the middle. 

Maybe isolating a `get_gecos` call? It will help to test this...

Reply:
Done!